### ISSUE TITLE
[Combobox] - Add ability to manually set aria-expanded attribute

### ISSUE THREAD

**Issue by mlimberg** [reach/reach-ui#447]
### Current Behavior

The `aria-expanded` attribute on the `Combobox` is set based on the status of its child `ComboboxInput`.

Manually setting the `aria-expanded` status on the `Combobox` as such has no effect

```javascript
<Combobox aria-expanded={toggledBoolean} />
``` 

This causes lowered scores in accessibility audits if a value is present in the input that you do not want to drop/expand the `ComboboxList`

### Desired Behavior

Ability to manually update the aria-expanded status based on provided logic:

```javascript
<Combobox aria-expanded={toggledBoolean} />
``` 

### Suggested Solution

Allow for `ariaExpanded` override prop

```javascript
<Combobox ariaExpanded={toggledBoolean} />
```

Or, pass a list of options that toggles the aria-expanded based on length of the array

```javascript
<Combobox options={comboboxOptions} />
```

### Who does this impact? Who is this for?

Autosuggests where a value may be present in the input on page load that is not intended to expand/open the ComboboxList

_Example: a location drop down that initially holds a value of "Near Me" based on geolocation. It does not trigger any suggestions and therefore aria-expanded should be "false", but instead it's true due to the input's value putting the input in a state of SUGGESTING_

### Describe alternatives you've considered

* overriding the aria-expanded, doesn't seem to work
* Set a placeholder to look like the value in certain cases (doesn't work with labels that shift position only when value is present)
* Adding a span element outside of the input to render the value without the input itself containing the value - 😢 

### Additional context

Please advise if there is another solution already available. I dug through the source code and did not see another way to tell the combobox it's not expanded when there is value in the input component

Thank you!

**Comment by chancestrickland** [comment]
Should be good in 0.8.2.
